cms/djangoapps/contentstore/views/component.py

```diff
@@ -16,6 +16,7 @@
 from xblock.core import XBlock
 from xblock.django.request import webob_to_django_response, django_to_webob_request
 from xblock.exceptions import NoSuchHandlerError
+from xblock.fields import Scope
```

Comment:
A bit of boyscouting here-- noticed that a section of code was referencing Scope when it wasn't imported.